atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -302,23 +302,25 @@ def set_group_manifest_info(self, extra, worker_metadatas):
             if not registries:
                 registries = self.workflow.push_conf.all_registries
             for registry in registries:
-                pullspec = "{0}/{1}@{2}".format(registry.uri, repo,
-                                                manifest_list_digests[0].v2_list)
+                manifest_list_digest = manifest_list_digests.get(repo)
+                if not manifest_list_digest:
```

Comment:
In which scenario would the key for the repo be defined in `manifest_list_digests` but not a value?

Reply:
Can't imagine one, its just a safety measure. Removed it for now, it doesn't seem to break the tests